nix/lib/pci/devices.nix

```diff
@@ -0,0 +1,74 @@
+{
+  virtio_scsi = {
+    vendorId = 6900; # 0x1af4
```

Comment:
At some point we should probably invert this list to make the scan faster i.e. not scanning the list multiple times.